execution-frameworks/Invoke-AtomicRedTeam/Invoke-AtomicRedTeam.ps1

```diff
@@ -0,0 +1,162 @@
+<#
+.SYNOPSIS
+This script will iterate over the Atomic Red Team yaml files, create objects for each test.
+The aim is to allow defenders to excercise MITRE ATT&CK Techniques to test defenses.
+
+
+Function: Invoke-AtomicRedTeam
+Author: Casey Smith @subTee
+License:  http://opensource.org/licenses/MIT
+Required Dependencies: powershell-yaml , Install-Module powershell-yaml #https://github.com/cloudbase/powershell-yaml
+Optional Dependencies: None
+Version: 1.0
+
+.DESCRIPTION
+Create Atomic Tests from yaml files described in Atomic Red Team. https://github.com/redcanaryco/atomic-red-team
+
+.EXAMPLE
+Convert Single Yaml File to Technique Object
+$T1117 = Get-AtomicTechnique -Path ..\..\atomics\T1117\T1117.yaml
+.EXAMPLE
+Get the Atomic Tests For A Given Technique
+Get-AtomicTest $T1117
+
+.NOTES
+This script converts Atomic Tests Expressed in YAML into PowerShell Objects.
+
+.LINK
+Blog: http://subt0x11.blogspot.com/2018/08/invoke-atomictest-automating-mitre-att.html
+Github repo: https://github.com/redcanaryco/atomic-red-team
+
+#>
+
+function Get-AtomicTechnique {
```

Comment:
Generally, PowerShell style is 4 *spaces* indentation. See this [style guide](https://github.com/PoshCode/PowerShellPracticeAndStyle/blob/master/Style-Guide/Code-Layout-and-Formatting.md) for more info.

Reply:
Fixed